### ISSUE TITLE
Tables embedded in white background needed

### ISSUE THREAD

**Issue by yih-wang** [patternfly/patternfly-next#1835]
We need tables in the expansion of data list in AMQ. In PF4, I haven't found a table in PF4 that can display in white background and don't occupy a whole row. 

I created an initial wireframe for such tables - 
<img width="1203" alt="Screen Shot 2019-05-17 at 15 58 37" src="https://user-images.githubusercontent.com/23331885/57912357-aa6d8600-78bc-11e9-9d7d-014e87c8c293.png">

**Comment by mcarrano** [comment]
@mcoker can you take a look at this? Could this be created by using one of our existing layouts or by placing the Messages and Clients elements within cards that sit inside the expansion?

**Comment by mcoker** [comment]
@mcarrano @yih-wang Yes, in this case, I would use one of the PF4 layouts inside of the expandable section of the table to create 2 columns, then in each column, place your header and graph/table content.

I mocked up an example here, using the `.pf-l-grid` (Grid) layout in the 3rd row. The base table in that example comes from our [table examples](https://pf4.patternfly.org/components/Table/examples/).

**Comment by mcoker** [comment]
@mcarrano re: borders, we don't currently have a table that has borders around the outer edges. The card would work, except the card has a box shadow, not a border. Do we want to add support for a border around a table, and/or a border with/without a shadow in the card?

**Comment by mcarrano** [comment]
That's a good question @mcoker . Before we commit to creating a new table variant, @kybaker and @mceledonia can you take a look at the mockup here  and let us know if there is a different way to handle this visually? Looks like when we use the table in a full screen view it's against a gray background so this isn't an issue. See https://www.patternfly.org/v4/documentation/core/demos/datatable  Thoughts?

**Comment by junezhang** [comment]
@mcarrano  @mcoker thanks for your quick response.
@kybaker @mceledonia, For the border issue, it makes sense without borders around the outer edges and without the background color for one table. But when there are two tables, if no border between the tables, users will be lost in the table list. 
Look like the below two pictures:
<img width="1216" alt="tablewithoutborders" src="https://user-images.githubusercontent.com/701009/58073711-e7e05500-7bd5-11e9-89aa-32b4c7e8127a.png">
<img width="1161" alt="tablewithborders" src="https://user-images.githubusercontent.com/701009/58073712-e878eb80-7bd5-11e9-963a-34ee7eec9345.png">
We would like to get your feedback.

**Comment by rachael-phillips** [comment]
Following up here on this conversation. As @mcarrano said, before we commit to creating a new table variant, @kybaker and @mceledonia can you take a look at the mockup here and let us know if there is a different way to handle this visually?
